modules/apps/commerce/commerce-product-content-web/src/main/java/com/liferay/commerce/product/content/web/internal/helper/CPContentHelperImpl.java

```diff
@@ -480,6 +489,45 @@ public String getImageURL(FileEntry fileEntry, ThemeDisplay themeDisplay)
 		return cpMedia.getURL();
 	}
 
+	public int getMiniumOrderQuantity(long cpDefinitionId) {
+		CPDefinitionInventory cpDefinitionInventory =
+			_cpDefinitionInventoryLocalService.
+				fetchCPDefinitionInventoryByCPDefinitionId(cpDefinitionId);
+
+		return cpDefinitionInventory.getMinOrderQuantity();
```

Comment:
What if it's null?

Reply:
I think it can't be null. But now if it's null we are returning 0 and in the UI the min quantity is not showed